ingestion/src/metadata/readers/dataframe/parquet.py

```diff
@@ -39,6 +43,57 @@ class ParquetDataFrameReader(DataFrameReader):
     from any source based on its init client.
     """
 
+    def _read_parquet_in_batches(self, parquet_file, batch_size: int = 10000):
```

Comment:
are we only allowing reading parquet in batches? Would we have the same error with other file types? also we might want typing around `parquet_file`